### DISCUSSION TITLE
Updating the theme

### DISCUSSION THREAD

**Discussion by kylereddoch** [sylhare/Type-on-Strap#365] [General]
Is there an easy way to update the theme?

**Comment by lg-nineoneninethreenine** [comment]
One way is to set the theme as a remote theme and updating via the gemfile, the other is to replace the files one-by-one. I have a method for this - copy the files you added/edited including the Jekyll config, then move it to a safe folder and then get the latest Type-On-Strap zipfile. Then copy everything in the new version and replace it in yours. Then copy the files kept safely(copy only, so that the files are kept as a backup) and replace them in your updated folder.

Seems so hard, but it isn't trust me.

**Comment by kylereddoch** [comment]
Okay so I have it already as a remote theme. So all I have to do in the gemfile is change the version number?Reason I’m asking is…all of a sudden, I’m getting build errors on GitHub when pushing any changes. But, the changes show up when I go to my site. Thanks

**Reply by sylhare** [reply]
That's intriguing, 
- Which version are you using?
- What type of error do you get?

If you are using it as a remote them you can remove the `.gemspec` and the folders like `_layouts`, `_includes` or `_sass` unless you have some custom pages
